src/test/resources/api/azure/users/_get/rsId.schema

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema" : "some-schema",
```

Comment:
remove that, if you don;t use

Reply:
it is quite hard to know, which parametres are required, so, following template is just a good way to start